### PR TITLE
refactor(full-planning): refactored full-planning directories

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-4130

## Description of change

refactored full-planning directories

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/routes/index.js [hunk 1]
```diff
@@ -17,6 +17,6 @@ router.use('/cookies', cookieRouter);
 router.use('/appellant-submission', checkDecisionDateDeadline, appellantSubmissionRouter);
 router.use('/eligibility', checkDecisionDateDeadline, eligibilityRouter);
 router.use('/your-planning-appeal', yourPlanningAppealRouter);
-router.use('/before-you-start', beforeYouStartRouter);
+router.use('/before-you-start', fullPlanningRouter);
```
Comment (nssdesikan) [inline packages/forms-web-app/src/routes/index.js:20]: Should this not be full-planning?

**Review by nssdesikan — CHANGES_REQUESTED** [review]
Nothing to do with this PR, but some test files are .spec and other .test. Don't know which one to follow

File: packages/forms-web-app/src/routes/full-planning/index.test.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+describe('routes/full-planning/index', () => {
+  beforeEach(() => {
+    jest.resetModules();
+
+    // eslint-disable-next-line global-require
+    require('./index');
+  });
+
+  afterEach(() => {
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply packages/forms-web-app/src/routes/full-planning/index.test.js:16]: Removed.

File: packages/forms-web-app/src/routes/full-planning/type-of-planning-application.test.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+jest.mock('../../validators/full-planning/type-of-planning-application');
 
 describe('routes/eligibility/planning-department', () => {
   beforeEach(() => {
     // eslint-disable-next-line global-require
-    require('../../../../src/routes/before-you-start/type-of-planning-application');
+    require('./type-of-planning-application');
   });
 
   afterEach(() => {
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply packages/forms-web-app/src/routes/full-planning/type-of-planning-application.test.js:17]: Removed.

File: packages/forms-web-app/src/routes/full-planning/use-a-different-service.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 const express = require('express');
```
[the remark this answers is not in the document]
Comment (swdpcomputing) [reply packages/forms-web-app/src/routes/full-planning/use-a-different-service.js:1]: Don't know. Doesn't matter for now.

File: packages/forms-web-app/src/routes/index.js (hunk 1 above)
Comment (swdpcomputing) [reply packages/forms-web-app/src/routes/index.js:20]: Yes, not sure how anything passed with that on.

**Review by nssdesikan — DISMISSED** [review]

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by nssdesikan — APPROVED** [review]

**Pull request merged**